projects/cdk/utils/dom/get-element-obscurers.ts

```diff
@@ -50,10 +51,12 @@ export function tuiGetElementObscures(element: Element): readonly Element[] | nu
     const nonNull = elements.filter(tuiIsPresent);
 
     if (!nonNull.length) {
-        return nonNull;
+        return nonNull as [];
```

Comment:
No need to typecast, we can just return `[]`

Reply:
fixed